src/vecgf2.c

```diff
@@ -2926,10 +2932,6 @@ Obj FuncTRANSPOSED_GF2MAT( Obj self, Obj mat)
   /* create new matrix */
   for (i = 1; i <= w; i++) {
     NEW_GF2VEC( row,  typ, l );
-
-    ptr=BLOCKS_GF2VEC(row);
-    for (n=1;n<=nrb;n++){*ptr=0;}
-
```

Comment:
This loops was buggy, repeatedly zeroing the very same memory cell (ptr is never incremented). But things worked anyway, because `NewBag` and hence `NEW_GF2VEC` gives us zero initialised allocations. So, I just removed it.

Reply:
agreed